### PR TITLE
Add cloud.aws_troubleshooting

### PR DESCRIPTION
**Pull request by alinabuzachis**
Depends-On: https://github.com/ansible/ansible-zuul-jobs/pull/1663

Add cloud.aws_troubleshooting

### CODE REVIEW

File: resources/ansible.yaml [hunk 1]
```diff
@@ -265,3 +265,5 @@ resources:
             zuul/include: []
         - goneri/vcenter-ldap-role-manager:
             zuul/include: []
+        - ansible-collections//cloud.aws_troubleshooting:
```
[the remark this answers is not in the document]
Comment (alinabuzachis) [reply resources/ansible.yaml]: @gravesm Thank you. Done!

**Comment by goneri** [comment]
You actually need to do two PR, one that target resources/ansible.yaml and after that, another PR to associate jobs on the new projects.

https://github.com/ansible/zuul-config#enable-the-github-application

**Comment by goneri** [comment]
I just pushed this PR to address the problem -> https://github.com/ansible/zuul-config/pull/457

**Comment by alinabuzachis** [comment]
> I just pushed this PR to address the problem -> #457

Thank you!

**Comment by goneri** [comment]
recheck

**Comment by goneri** [comment]
> > I just pushed this PR to address the problem -> #457
> 
> Thank you!

De nada, we literally all do this mistake :-).

**Pull request merged**